krod/interfaces/cli.py

```
@@ -64,17 +66,16 @@ def process_command(self, command: str) -> bool:
 
         elif command.startswith('vector_store'):
             try:
-                import shlex
+                import shlex, click
```

Flake8 (7.2.0)

[error] multiple imports on one line

(E401)